### DISCUSSION TITLE
multiple i2c master instances ?

### DISCUSSION THREAD

**Discussion by tiisaidipjp** [felias-fogg/SoftI2CMaster#72] [General]
Hi
Is it possible to use multiple i2c master instances ?

Example
pin D1,D2 for I2c master A
Pin D4,D5  for  I2c master B
Pin D6,D7  for I2c master C

**Comment by felias-fogg** [comment]
This does not work with SoftI2CMaster, but with SlowSoftI2CMaster: https://github.com/felias-fogg/SlowSoftI2CMaster
